src/testlib/include/testlib/helpers.hpp

```diff
@@ -40,6 +43,21 @@ inline std::string toString(const YAML::Node& node) {
     return std::string{out.c_str()};
 }
 
+inline std::string toString(int i) {
+    return std::to_string(i);
+}
+
+// TODO: this is duped with MongoTestFixture
```

Comment:
`MongoTestFixture` didn't work out very well. The mongo client really wants to be part of the actor and not the fixture. We should probably kill it and move `static makeApmCallback()` and `static connectionUri()` to a helper library.

Helper libraries seem to play nicer since we're using catch2's `TEST_CASE` instead of `TEST_CASE_METHOD` style of testing.

Reply:
Ya I've done that in the real review - coming up!